test/models/position_test.rb

```diff
@@ -3,16 +3,12 @@
 class PositionTest < Minitest::Test
 
   def setup
-    @position = position.dup
-    @position.save!
-  end
-
-  def teardown
```

Comment:
I think we should keep the `teardown` method, and make sure it destroys the run it creates. Otherwise, our test database will fill up with Runs.